WordPress/Classes/Services/MediaImageService.swift

```diff
@@ -61,124 +80,58 @@ final class MediaImageService: NSObject {
         }
     }
 
-    @MainActor
-    private func _imageURL(for media: Media, preferredSize: CGSize) async throws -> URL {
-        let mediaID = TaggedManagedObjectID(saved: media)
-        let exporter = makeThumbnailExporter(for: media, preferredSize: preferredSize)
-
-        // Check if there is already an exported thumbnail available
-        if let identifier = media.localThumbnailIdentifier, let imageURL = await getLocalThumbnailURL(for: identifier, exporter: exporter) {
-            return imageURL
-        }
-
-        // Downloads the remote thumbnail for the asset.
-        @MainActor func getRemoteThumbnail() async throws -> URL {
-            let image = try await downloadThumbnail(forMedia: media, preferredSize: preferredSize)
-            let (identifier, export) = try await exporter.exportThumbnail(forImage: image)
-            try await saveLocalIdentifier(identifier: identifier, for: mediaID)
-            return export.url
-        }
-
-        // If the asset is available locally, export thumbnails from the local asset
-        if let localAssetURL = media.absoluteLocalURL,
-           exporter.supportsThumbnailExport(forFile: localAssetURL) {
-            let (identifier, export) = try await exporter.exportThumbnail(forFileURL: localAssetURL)
-            try await saveLocalIdentifier(identifier: identifier, for: mediaID)
-            return export.url
-        }
-
-        // If the Media item is a video and has a remote video URL, try and export from the remote video URL.
-        if media.mediaType == .video, let videoURL = media.remoteURL.flatMap(URL.init) {
-            do {
-                let (identifier, export) = try await exporter.exportThumbnail(forVideoURL: videoURL)
```

Comment:
Not sure if it's needed. Maybe for self-hosted sites?
I have a note to test it later when I start adding proper video support for cells.